obitamockbank100.catalog.obita.tech/resources/api.yaml

```diff
@@ -0,0 +1,530 @@
+      responses:
+        '201':
+          description: Transaction submitted
+        '400':
+          description: 'invalid input, object invalid'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Transaction'
+        description: Bank transaction submitted
+  /beneficiary:
+    post:
+      summary: Create a beneficiary
+      operationId: createBeneficiary
+      description: Create a beneficiary
+      responses:
+        '201':
+          description: Beneficiary created
+        '400':
+          description: 'invalid input, object invalid'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Beneficiary'
+        description: Beneficiary created
+    delete:
+      summary: Delete a beneficiary
+      operationId: deleteBeneficiary
+      description: Delete a beneficiary
+      parameters:
+        - in: query
+          name: beneficiaryAccountNumber
+          description: beneficiary account number
+          required: true
+          schema:
+            type: string
+        - in: query
+          name: linkedAccountNumber
+          description: linked account number
+          required: true
+          schema:
+            type: string
+      responses:
+        '201':
+          description: Beneficiary deleted
+        '400':
+          description: 'invalid input, object invalid'
+  /standing-order:
+    post:
+      summary: Create a standing order
+      operationId: createStandingOrder
+      description: Create a standing order
+      responses:
+        '201':
+          description: Standing order created
+        '400':
+          description: 'invalid input, object invalid'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Standing-Order'
+        description: Standing order created
+  /scheduled-payment:
+    post:
+      summary: Create a scheduled payment
+      operationId: createScheduledPayment
+      description: Create a scheduled payment
+      responses:
+        '201':
+          description: Scheduled payment created
+        '400':
+          description: 'invalid input, object invalid'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Transaction'
+        description: Scheduled payment created
+  /direct-debit:
+    post:
+      summary: Create a direct debit
+      operationId: createDirectDebit
+      description: Create a direct debit
+      responses:
+        '201':
+          description: Direct debit created
+        '400':
+          description: 'invalid input, object invalid'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Direct-Debits'
+        description: Direct debit created
+  /balance:
```

Comment:
I don't think we need a seperate endpoint to create balance. Balance should be calculated and populated as a part of other usecases. eg: do a cash transfer, pay a bill etc